README.md

```diff
@@ -75,7 +83,15 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+var pizza = {
+  temp: 70,
+  bake: function cook() {
+      pizza.temp = 300;
+  }
+ }
+
+pizza.bake();
+console.log(pizza);
 ```
```

Comment:
You're not allowed to reference `pizza` inside of the object! Instead, make use of the `this` keyword.

```js
var pizza = {
  temperature: 70,
  bake: function(){
    this.temperature = 300;
  }
}
```